stdlib/public/core/CharacterProperties.swift

```diff
@@ -32,7 +32,7 @@ extension Character {
   ///
   ///     let chars: [Character] = ["a", " ", "™"]
   ///     for ch in chars {
-  ///         print(ch, "-->", ch.properties.numericValue)
+  ///         print(ch, "-->", ch.asciiValue)
   ///     }
   ///     // a --> 97
```

Comment:
The accessor change also results in an `Optional` value being printed.

Reply:
`.properties.numericValue` used to be optional as well.

Reply:
The style of showing outputs of all snippets in this file doesn't show exactly what is printed.
Do you think it should be changed to following?
```
/// Prints:
/// a --> Optional(97)
///   --> Optional(32)
/// ™ --> nil
```

Reply:
Yes

Reply:
Fixed